src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js

```diff
@@ -177,7 +195,7 @@ define(['sulumedia/model/media'], function(Media) {
                 },
                 {
                     el: this.$find(constants.datagridSelector),
-                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC',
+                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC&locale=' + this.locale,
```

Comment:
As said, it's not really your issue, just asked you to change it, if you already have changed this line anyway :smiley: